apps/example-app/babel.config.js

```diff
@@ -11,7 +11,8 @@ module.exports = function (api) {
   api.cache(true)
   return {
     assumptions: {
-      "setPublicClassFields": true,
+      setPublicClassFields: true,
+      privateFieldsAsSymbols: true,
```

Comment:
What are these for?

Reply:
Ok I see it's for the following console warnings, maybe let's note that in a comment?

```
[transform-private-methods]: You are using the "loose: true" option and you are explicitly setting a value for the "setPublicClassFields" assumption. The "loose" option can cause incompatibilities with the other class features plugins, so it's recommended that you replace it with the following top-level option:
transform[stderr]: 	"assumptions": {
transform[stderr]: 		"setPublicClassFields": true,
transform[stderr]: 		"privateFieldsAsSymbols": true
transform[stderr]: 	}
```

Reply:
we spent a little time trying to track down what's going on with  babel and why we might even need to set exceptional config around assumptions / the class property transform, kind of came up empty. @morganick @joshuayoes is this something you have context on, since you were the last to look at this config?

Reply:
These settings are from the initial warning that said to set it to this value. It did not appear to help reduce the number of warnings. I unfortunately was not able to get to the end of this 🐰 🕳️ . It also did not break the build. I have a feeling the issue is subtle and isn't *currently* causing issue that we're away of.

Reply:
Yeah it's probably fine to merge, I'm just wondering if we might be carrying config that doesn't matter. Something else to follow up on later.